site/_includes/macros/render-type.njk

```diff
@@ -206,8 +206,16 @@
     {% set iconType = type.type %}
   {% endif %}
 
+   
+  {% set description = "" %}
+  {% for enum in node._enums %}
+    {% if enum.value == type.value %}
+      {% set description = enum.description %}
+    {% endif %}
+  {% endfor %}
+
   {# Note that we avoid whitespace below, and instead set margins in CSS. #}
-  <p class="code-sections__icon code-sections__icon--{{ iconType }}">{{ renderSingleType(type, node) | trim | safe }}
+  <p class="code-sections__icon code-sections__icon--{{ iconType }}">{{ renderSingleType(type, description) | trim | safe }}
```

Comment:
Looking at https://pr-6201-static-dot-dcc-staging.uc.r.appspot.com/docs/extensions/reference/offscreen/#type-Reason it seems as if the descriptions are markdown and could benefit from running through `| mdInline` 🙂